system-addon/content-src/components/Search/Search.jsx

```diff
@@ -34,11 +34,11 @@ class Search extends React.Component {
 
   render() {
```

Comment:
Please add comments here about which of these attributes have magic side-effects and what those are, so that they don't get inadvertently changed in the future.